README.mkdn

```diff
@@ -152,9 +152,9 @@ application described by the TOML file.
 - `cargo xtask dist app/demo-stm32f4-discovery/app.toml` - stm32f4-discovery
 - `cargo xtask dist app/demo-stm32f4-discovery/app-f3.toml` - stm32f3-discovery
 - `cargo xtask dist app/lpc55xpresso/app.toml` - lpcxpresso55s69
-- `cargo xtask dist app/demo-stm32g0-nucleo/app-g031.toml` - stm32g031
-- `cargo xtask dist app/demo-stm32g0-nucleo/app-g070.toml` - stm32g070
-- `cargo xtask dist app/demo-stm32g0-nucleo/app-g0b1.toml` - stm32g0b1
+- `cargo xtask dist app/demo-stm32g0-nucleo/app-g031.toml` - stm32g031-nucleo
+- `cargo xtask dist app/demo-stm32g0-nucleo/app-g070.toml` - stm32g070-nucleo
+- `cargo xtask dist app/demo-stm32g0-nucleo/app-g0b1.toml` - stm32g0b1-nucleo
```

Comment:
Naming nit: should `nucleo` be at the start or end?  We have both `stm32g0b1-nucleo` and `nucleo-ih743zi2` board names.

Reply:
It appears that the vendor consistently puts nucleo at the front of the name, so, that's an argument for consistently doing that too.